pkg/agent/route/interfaces.go

```diff
@@ -38,6 +38,13 @@ type Interface interface {
 	// It should do nothing if the routes don't exist, without error.
 	DeleteRoutes(podCIDR *net.IPNet) error
 
+	// AddClusterIPRoute adds a route for a Service. It redirects ClusterIP traffic from the Windows host network
+	// to AntreaProxy.
+	AddClusterIPRoute(svcIP net.IP) error
+
+	// DeleteServiceRoutes deletes route for a Service IP when dealing with ClusterIP Service with antrea-proxy.
+	DeleteServiceRoutes(svcIP net.IP) error
```

Comment:
DeleteServiceRoutes() also handles non-ClusterIP Services? If not, why it is named differently from AddClusterIPRoute()?

If it just deletes a single route, remove "s" from Routes.

Reply:
No, it's for ClusterIP Service. Updated to `DeleteClusterIPRoute`.